src/pyggdrasil/scripts/cell_simulation.py

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Generate cell mutation matrices,
+by generating random trees and sampling cell attachments
+with noise if error rates are non-zero.
+
+As per definition in the SCITE Jahn et al. 2016.
+"""
+
+import argparse
+import jax.random as random
+import numpy as np
+from jax.random import PRNGKeyArray
+import json
+import os
+
+from pyggdrasil.tree import TreeNode
+from pyggdrasil.serialize._to_json import serialize_tree_to_dict
+import pyggdrasil.tree_inference._simulate as sim
```

Comment:
Is this the correct/pretty way to import these still private functions from a script like this? - Or shall we make them accessible via their respective `__initi__.py`

Reply:
You've raised a very good point – we  should perfectly use the public API only, exporting the used functions to the `__init__.py` (right now everything except from `TreeNode` seems to live in private submodules).

Generally, the reasoning is as this:
- It's easier to add a function than to remove it from the codebase (so that the users don't have errors when they update the package).
- But if we don't have a public API and use private API anyway, then the purpose of having private API doesn't really exist.

I think we can proceed with adding functions that are necessary in the scripts to the `__init__.py` and the API will form organically. (At some point we'll need to probably trim it down/refactor, but let's not think about it now).